### PR TITLE
GH-1537 AM Chart Balloon Text Issue

### PR DESCRIPTION
**Pull request by GordonSmith**
Use correct series from graph object.

Fixes GH-1537

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo @jchambers-ln Please Review

File: src/amchart/CommonSerial.js [hunk 1]
```diff
@@ -506,13 +506,10 @@
...
-            //var balloonText = d.category + ", " + context.columns()[d.graph.columnIndex+1]  + ": " + context.data()[d.index][d.graph.columnIndex+1];
-            if (d.graph.type === "line") {
-                return d.category + ", " + context.columns()[d.graph.index + 1]  + ": " + context.data()[d.index][d.graph.index + 1];
-            } else {
-                return d.category + ", " + context.columns()[d.graph.columnIndex + 1]  + ": " + context.data()[d.index][d.graph.columnIndex + 1];   
-            }
+        gObj.balloonFunction = function(item, graph) {
+            var key = graph.valueField;
+            var data = graph.chart.dataProvider;
+            return item.category + ", " + key + ": " + data[item.index][key];
```
Comment (jchambers-ln) [inline src/amchart/CommonSerial.js]: is it possible for those indexes not to exist in data?

Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: I don't believe so (only if there was no bar chart displayed) - I tested on as many samples as I could and it looked good.  
There is an identical issue with click I just spotted, so will open an issue for that as well.

**Comment by mlzummo** [comment]
looks like it should be ok

**Pull request merged**